app/models/flood_risk_engine/organisation.rb

```diff
@@ -12,5 +12,11 @@ class Organisation < ActiveRecord::Base
       other: 5,
       unknown: 6
     }
+
+    has_one :primary_address, -> { where(address_type: Address.address_types["primary"]) },
+            class_name: "FloodRiskEngine::Address",
+            as: :addressable,
+            dependent: :restrict_with_exception
+
   end
 end
```

Comment:
Can an organisation have >1 address? If not we could lose the `primary_` prefix maybe (implies there may be a secondary one etc)?